### PR TITLE
fix: aka phond and mobile

### PR DESCRIPTION
**Pull request by 770menahem**

### CODE REVIEW

File: src/gen/ad.gen.ts [hunk 1]
```diff
@@ -1,21 +1,26 @@
-import { adUser } from '../types/types';
+import { adUser, user } from '../types/types';
 import faker from 'faker';
 import dataTypes from '../lists/dataOption';
 import utils from '../utils/utils';
 
-export function createAdUser(firstName: string, lastName: string, mi: string) {
+export function createAdUser(user: user | undefined = undefined) {
```
Comment (Eli-BB8) [inline src/gen/ad.gen.ts:6]: user?

File: src/gen/ad.gen.ts [hunk 2]
```diff
@@ -1,21 +1,26 @@
...
   const job = faker.name.jobTitle();
   const sAMAccountName = faker.internet.email().split('@')[0];
+  const firstName = (user ? user.firstName : faker.name.firstName()) as string;
+  const lastName = (user ? user.lastName : faker.name.lastName()) as string;
+  const mi = user ? user.mi : utils.generateID();
 
   let ad: adUser = {
     KfirstName: firstName,
     guName: firstName,
```
Comment (Eli-BB8) [inline src/gen/ad.gen.ts:15]: check

File: src/gen/aka.gen.ts [hunk 3]
```diff
@@ -1,18 +1,15 @@
...
-    .toISOString();
-  const createdAt = faker.date
-    .between(faker.date.past(1), takenAt)
-    .toISOString();
-  const updatedAt = faker.date
-    .between(faker.date.past(1), createdAt)
-    .toISOString();
+export function createAkaUser(
+  tzs: string = utils.generateID(),
+  mis: string = utils.generateNumberAsString()
```
Comment (Eli-BB8) [inline src/gen/aka.gen.ts:8]: remove s from name

File: src/gen/city.gen.ts [hunk 4]
```diff
@@ -1,100 +1,149 @@
...
+  return cityUser;
+}
+
+export function createCityUserGU() {
+  const firstName = faker.name.firstName();
+  const lastName = faker.name.lastName();
+  const domains =
+    utils.generateNumberAsString(0, 3) === '0'
+      ? []
+      : (utils.randomArrFromArr(dataTypes.CITY_DOMAINS as unknown as any[]) as any);
```
Comment (Eli-BB8) [inline src/gen/city.gen.ts:92]: only one domain

File: src/types/types.ts [hunk 5]
```diff
@@ -1,6 +1,11 @@
 import dataEnum from '../lists/dataOption';
-
-export type esUser = {
+export interface user {
```
Comment (Eli-BB8) [inline src/types/types.ts:2]: Consider using type instead of interface

Comment (770menahem) [reply src/types/types.ts:2]: inteface can extends type not

File: src/gen/ad.gen.ts (hunk 2 above)
Comment (770menahem) [reply src/gen/ad.gen.ts:15]: as old karting mock

**Pull request merged**